tests/python/pants_test/base_test.py

```diff
@@ -216,7 +216,7 @@ def setUp(self):
 
     self.build_root = os.path.realpath(mkdtemp(suffix='_BUILD_ROOT'))
     self.subprocess_dir = os.path.join(self.build_root, '.pids')
-    self.addCleanup(safe_rmtree, self.build_root)
+    #self.addCleanup(safe_rmtree, self.build_root)
```

Comment:
Ooops, no, that was a relic of my frantic debugging efforts. 

This is why we do code reviews! :)